image-builder-cli.spec

```diff
@@ -51,6 +51,8 @@ BuildRequires:  btrfs-progs-devel
 # BUNDLE_END
 %endif
 
+Requires:       osbuild
```

Comment:
Do we also need osbuild-depsolve-dnf ?

Reply:
You're too quick :D

Reply:
Ok, now everything is listed including the minimal osbuild version we already had.